client/src/charts/canada/canada.js

```diff
@@ -57,7 +58,7 @@ class CanadaGraphBarLegend extends React.Component {
 
     return (
       <Fragment>
-        <p className="mrgn-bttm-0 mrgn-tp-0 nav-header centerer">
+        <p className="mrgn-bttm-0 mrgn-tp-0 title-p--canada centerer">
```

Comment:
Trying to follow the BEM naming convention, I ended up with title-p--canada. Open to changing this into something that fits more.

Reply:
Should switch to using `LegendContainer` in this file now, the title can just be passed in as a prop to that (so don't need `title-p--canada` either).